Re-add 2 Fedora hard dependencies (rebased)

linux-modder opened the pull request.

linux-modder committed:
Re-add 2 Fedora hard dependencies (rebased)

zetok commented on INSTALL.md:
```diff
@@ -151,7 +151,7 @@ The following steps assumes that you cloned the repository at "/home/user/qTox".
 
 <a name="other-deps" />
-### GCC, Qt, FFmpeg, OpanAL Soft and qrencode
+### GCC, Qt, FFmpeg, OpanAL Soft and qrencode ffmpeg-devel
```
this change isn't needed

zetok commented on INSTALL.md:
````diff
@@ -183,7 +183,7 @@ sudo apt-get install build-essential qt5-qmake qt5-default qttools5-dev-tools li
 ```bash
 sudo dnf group install "Development Tools"
-sudo dnf install qt-devel qt-doc qt-creator qt5-qtsvg qt5-qtsvg-devel openal-soft-devel libXScrnSaver-devel qrencode-devel
+sudo dnf install qt-devel qt-doc qt-creator qt5-qtsvg qt5-qtsvg-devel openal-soft-devel libXScrnSaver-devel qrencode-devel qtsingleapplication  ffmpeg-devel
````
could you use 1 space, instead of 2?

linux-modder committed:
made changes as requested ( as I understood them)

- Removed  mention of  ffmpeg-devel in `other-deps`  in Fedora section
- Fixed spacing  error (whitespace)

linux-modder commented:
@zetok, that look more  like it  should now?

zetok commented:
@linux-modder yes. Could you made that change in 1 commit (current PR has already 2)

linux-modder commented:
Thought I had.  sure can
